docs/src/components/TwoColumn/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import styles from "./styles.module.scss"
+export const TwoColumn = ({ children }) => {
+    return(
+        <div className={styles.twoColumn}>
```

Comment:
can we take the size from params,
like if we want to add the image on left side!

Reply:
Can we pls give a different class name here, it might confuse the next developer who will takeover this code from us.